configs/advanced/load_guard.yaml

```diff
@@ -0,0 +1,56 @@
+# Example configuration for Caching Layer Load Guard
+# This file demonstrates the new load guard configuration parameters introduced in Milvus 2.6.0
+
+queryNode:
+  # Load guard configuration for caching layer resource management
+  loadGuard:
+    # Cache ratio: proportion of resources reserved for evictable cache data
+    # Range: 0.0 - 1.0
+    # Default: 0.2 (20% of resources reserved for cache)
+    # - 0.0: No cache, all data must fit in memory (no eviction benefit)
```

Comment:
0.0 means before loading inevictable data, we can use memory/disk to cache swap evictable data, but inevictable data may eventually be loaded and use up node resources, causing evictable data to not be able to be loaded at all, which is kind of expected behavior if the user choose to configure as such